float64 phiRoll
float64 thetaPitch
float64 psiYawRate
float64 thrust

int8 controlFlag
